journald/README.md

```diff
@@ -0,0 +1,138 @@
+# Agent Integration: journald
+
+## Overview
+
+Systemd-journald is a system service that collects and stores logging data. 
+It creates and maintains structured, indexed journals based on logging information that is received from a variety of sources.
+
+## Setup
+
+### Installation
+
+The journald check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+Journal files are, by default, owned and readable by the systemd-journal system group. To start collecting your journal logs, you need to:
+
+1. [Install the Agent][3] on the instance running the journal
+2. Add the `dd-agent` user to the `systemd-journal` group by running:
+    ```text
+     usermod -a -G systemd-journal dd-agent
+    ```
+
+{{< tabs >}}
+{{% tab "Host" %}}
+
+To configure this check for an Agent running on a host:
+
+Edit the `journald.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][4] to start collecting logs.
+
+#### Log collection
+
+Collecting logs is disabled by default in the Datadog Agent, you need to enable it in `datadog.yaml` with:
+
+```yaml
+logs_enabled: true
+```
+
+Then add this configuration block to your `journald.d/conf.yaml` file to start collecting your Logs:
+
+```yaml
+logs:
+    - type: journald
+      container_mode: true
+```
+
+To fill `source` and `service` attributes, the Agent collects `SYSLOG_IDENTIFIER` , `_SYSTEMD_UNIT` and `_COMM`and set them to the first non empty value. In order to take advantage of the integration pipelines, Datadog recommends setting the `SyslogIdentifier` parameter in the `systemd` service file directly, or in a `systemd` service override file. Their location depends on your distribution, but you can find the location of the `systemd` service file by using the command `systemctl show -p FragmentPath <unit_name>`.
+
+**Note**: With Agent 7.17+, if `container_mode` is set to `true`, the default behavior changes for logs coming from docker containers. The `source` attribute of your logs is automatically set to the corresponding short image name of the container instead of simply `docker`.
```

Comment:
```suggestion
**Note**: With Agent 7.17+, if `container_mode` is set to `true`, the default behavior changes for logs coming from Docker containers. The `source` attribute of your logs is automatically set to the corresponding short image name of the container instead of simply `docker`.
```